Formula/rbw.rb

```diff
@@ -0,0 +1,18 @@
+class Rbw < Formula
+  desc "Unofficial command-line client for Bitwarden"
+  homepage "https://github.com/doy/rbw"
+  url "https://github.com/doy/rbw/archive/0.5.2.tar.gz"
+  sha256 "5ab3c569ea55eeee6e70130c3d1a3399ac0529b666e2ba12264c1a7b0b5dbc3c"
+  license "MIT"
+
+  depends_on "rust" => :build
+  depends_on "pinentry"
```

Comment:
```suggestion
  depends_on "rust" => :build
  depends_on "openssl@1.1"
  depends_on "pinentry"
```
You need this to address the following errors:
```
==> brew linkage --test rbw
==> FAILED
Broken dependencies:
  /usr/local/opt/openssl@1.1/lib/libcrypto.1.1.dylib (openssl@1.1)
  /usr/local/opt/openssl@1.1/lib/libssl.1.1.dylib (openssl@1.1)
```
```
==> /usr/local/Cellar/rbw/0.5.2/bin/rbw --version
dyld: Library not loaded: /usr/local/opt/openssl@1.1/lib/libssl.1.1.dylib
  Referenced from: /usr/local/Cellar/rbw/0.5.2/bin/rbw
  Reason: image not found
```